code/controllers/subsystem/statpanel.dm

```diff
@@ -92,11 +92,14 @@ SUBSYSTEM_DEF(statpanels)
 			return
 
 /datum/controller/subsystem/statpanels/proc/set_status_tab(client/target)
+#if MIN_COMPILER_VERSION > 515
+	#warn 516 is most certainly out of beta, remove this beta notice if you haven't already
```

Comment:
Probably not, we like to use betas a lot :)

Reply:
well then y'all can remove this `#warn` after setting `MIN_COMPILER_VERSION` to 516